src/app/components/header/header.component.ts

```diff
@@ -1,17 +1,22 @@
 import { Component, OnInit } from '@angular/core';
 import { CATEGORIES } from '@mocks/mock-categories';
 import { CartService } from '@shared/services/cart.service';
+import { SignInComponent } from '../sign-in/sign-in.component';
+import { AuthService } from '@shared/services/auth.service';
+import { MatDialog } from '@angular/material/dialog';
+import { UserInterface } from '@shared/interfaces/user-interface';
 
 @Component({
     selector: 'app-header',
     templateUrl: './header.component.html',
     styleUrls: ['./header.component.scss']
 })
 export class HeaderComponent implements OnInit {
+    public categories: {}[] = CATEGORIES;
 
-    categories: {}[] = CATEGORIES;
-
-    constructor(private cartService: CartService) {
+    constructor(private cartService: CartService,
+                public authService: AuthService,
+                private dialog: MatDialog) {
```

Comment:
align it normaly

Reply:
What do you mean? One line?

Reply:
Or like  here?
![image](https://user-images.githubusercontent.com/53089462/102976775-bdb9ec00-450a-11eb-9586-ff0083abfed0.png)

Reply:
it should be like this:
```ts
constructor(
  private cartService: CartService,
  private authService: AuthService,
  private dialog: MatDialog
) {}
```